js/main.js

```diff
@@ -1,8 +1,58 @@
 var $start_button = $(".start_button");
+var fall_time = 4000
+var interval_between_cows = 3000
 
-// Switch between main page to play page
+/**
+ * iterates core functions
+ */
+
+function game_loop() {
+    var column = Math.floor(Math.random() * 6)  // Random number from 0 to 5.
+    increase_speed()
+    create_cow(column, fall_time)
+    setTimeout(game_loop, interval_between_cows)
+}
+
+/**
+ * creates new cows
+ * @param column_number number column from 0-5
+ * @param fall_time  number time it takes for cow to fall
+ */
+
+function create_cow(column_number, fall_time) {
+    var horizontal_position = 5 + (110 * column_number)
+
+    var $container_div = $("<div class='cow_target_container absolute'>" +
```

Comment:
what does the absolute class do??

Reply:
As discussed later, absolute class was pre-existing.